### ISSUE TITLE
pip-autoremove crashes if any of package misses some dependencies

### ISSUE THREAD

**Issue by bhoppi** [invl/pip-autoremove#9]
for example, on my machine, I installed tensorflow==1.2.1, which depends on bleach==1.5.0, but bleach==2.0.0 installed (the versions of the packages are all controled by system package manager). When I run "pip-autoremove -l", it crashes and gives the output:

Traceback (most recent call last):
  File "/home/gfioser/.local/bin/pip-autoremove", line 11, in <module>
    sys.exit(main())
  File "/home/gfioser/.local/lib/python3.6/site-packages/pip_autoremove.py", line 107, in main
    list_dead(args)
  File "/home/gfioser/.local/lib/python3.6/site-packages/pip_autoremove.py", line 29, in list_dead
    graph = get_graph()
  File "/home/gfioser/.local/lib/python3.6/site-packages/pip_autoremove.py", line 92, in get_graph
    for req in requires(dist):
  File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 556, in get_distribution
    dist = get_provider(dist)
  File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 430, in get_provider
    return working_set.find(moduleOrReq) or require(str(moduleOrReq))[0]
  File "/usr/lib/python3.6/site-packages/pkg_resources/__init__.py", line 718, in find
    raise VersionConflict(dist, req)
pkg_resources.VersionConflict: (bleach 2.0.0 (/usr/lib/python3.6/site-packages), Requirement.parse('bleach==1.5.0'))

Can you just ignore the missing dependencies, or at least ignore the version differences? Thanks.

**Comment by gangmax** [comment]
What I got is the following error when running "pip-autoremove":

``` bash
~> pip-autoremove douban.fm
Traceback (most recent call last):
  File "/home/gang/.pyenv/versions/3.6.1/bin/pip-autoremove", line 11, in <module>
    sys.exit(main())
  File "/home/gang/.pyenv/versions/3.6.1/lib/python3.6/site-packages/pip_autoremove.py", line 109, in main
    autoremove(args, yes=opts.yes)
  File "/home/gang/.pyenv/versions/3.6.1/lib/python3.6/site-packages/pip_autoremove.py", line 21, in autoremove
    dead = list_dead(names)
  File "/home/gang/.pyenv/versions/3.6.1/lib/python3.6/site-packages/pip_autoremove.py", line 29, in list_dead
    graph = get_graph()
  File "/home/gang/.pyenv/versions/3.6.1/lib/python3.6/site-packages/pip_autoremove.py", line 92, in get_graph
    for req in requires(dist):
  File "/home/gang/.pyenv/versions/3.6.1/lib/python3.6/site-packages/pkg_resources/__init__.py", line 557, in get_distribution
    dist = get_provider(dist)
  File "/home/gang/.pyenv/versions/3.6.1/lib/python3.6/site-packages/pkg_resources/__init__.py", line 431, in get_provider
    return working_set.find(moduleOrReq) or require(str(moduleOrReq))[0]
  File "/home/gang/.pyenv/versions/3.6.1/lib/python3.6/site-packages/pkg_resources/__init__.py", line 719, in find
    raise VersionConflict(dist, req)
pkg_resources.VersionConflict: (six 1.10.0 (/home/gang/.pyenv/versions/3.6.1/lib/python3.6/site-packages), Requirement.parse('six==1.8.0'))

# After I change package "six" version from "1.10.0" to "1.8.0", I got the following error:
~> pip-autoremove douban.fm
Traceback (most recent call last):
  File "/home/gang/.pyenv/versions/3.6.1/bin/pip-autoremove", line 11, in <module>
    sys.exit(main())
  File "/home/gang/.pyenv/versions/3.6.1/lib/python3.6/site-packages/pip_autoremove.py", line 109, in main
    autoremove(args, yes=opts.yes)
  File "/home/gang/.pyenv/versions/3.6.1/lib/python3.6/site-packages/pip_autoremove.py", line 21, in autoremove
    dead = list_dead(names)
  File "/home/gang/.pyenv/versions/3.6.1/lib/python3.6/site-packages/pip_autoremove.py", line 29, in list_dead
    graph = get_graph()
  File "/home/gang/.pyenv/versions/3.6.1/lib/python3.6/site-packages/pip_autoremove.py", line 92, in get_graph
    for req in requires(dist):
  File "/home/gang/.pyenv/versions/3.6.1/lib/python3.6/site-packages/pkg_resources/__init__.py", line 557, in get_distribution
    dist = get_provider(dist)
  File "/home/gang/.pyenv/versions/3.6.1/lib/python3.6/site-packages/pkg_resources/__init__.py", line 431, in get_provider
    return working_set.find(moduleOrReq) or require(str(moduleOrReq))[0]
  File "/home/gang/.pyenv/versions/3.6.1/lib/python3.6/site-packages/pkg_resources/__init__.py", line 719, in find
    raise VersionConflict(dist, req)
pkg_resources.VersionConflict: (six 1.8.0 (/home/gang/.pyenv/versions/3.6.1/lib/python3.6/site-packages), Requirement.parse('six>=1.9.0'))
```

**Comment by jorgeorpinel** [comment]
I was getting this same problem with the **six** package, which is a common dependency. In my case, I'm on Mac OS 10.13.3 with stock Python 2.7.10 and six 1.4.0 is pre-installed in `/System/Library/Frameworks/Python.framework/Versions/2.7/Extras/lib/python` ...

At some point I managed to update six with `sudo easy_install` which put it separately in `/Users/drbidibombom/Library/Python/2.7/lib/python/site-packages`

**Comment by louismartin** [comment]
Same problem here

**Comment by tresni** [comment]
Should be addressed by the merge of #42 and the new release, please close if that resolves your issue.
